packages/react-core/src/core/providers/thirdweb-provider.tsx

```diff
@@ -178,7 +180,7 @@ const ThirdwebSDKProviderWrapper = <TChains extends Chain[]>({
 }: React.PropsWithChildren<
   Omit<ThirdwebSDKProviderProps<TChains>, "signer">
 >) => {
-  const signer = useWalletSigner();
+  const signer = useThirdwebWallet()?.signer;
```

Comment:
I liked the `useSigner` hook :(

Reply:
@iketw We still have it. I just removed it's duplicate which was only being used a single time internally